### PR TITLE
:package: Adds configuration for distribution

### PR DESCRIPTION
**Pull request by jmelberg-okta**
- Updates `group` to be `com.okta.android`
- Sets package name to `appauth-android`
- Adds `jfrog` configuration
- Sets version to `0.1.0`

Proposed Maven snippet:
```xml
<dependency>
  <groupId>com.okta.android</groupId>
  <artifactId>appauth-android</artifactId>
  <version>0.1.0</version>
  <type>pom</type>
</dependency>
```

Gradle:
```
compile 'com.okta.android:appauth-android:0.1.0'
```

Ivy:
```html
<dependency org='com.okta.android' name='okta-appauth' rev='0.1.0'>
  <artifact name='appauth-android' ext='pom' ></artifact>
</dependency>
```

Resolves:
 - #4, #5 
- [OKTA-140964](https://oktainc.atlassian.net/browse/OKTA-140964)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,9 +6,8 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:2.3.3"
-
-        // NOTE: Do not place your application dependencies here; they belong
```
Comment (wdawson) [inline build.gradle]: don't do this here. The note is for a reason :)

File: library/build.gradle [hunk 2]
```diff
@@ -1,8 +1,11 @@
 apply plugin: "com.android.library"
+apply plugin: "com.jfrog.bintray"
+apply plugin: "com.github.dcendents.android-maven"
 
 apply from: "${rootDir}/gradle/android-common.gradle"
 
-group = "com.okta"
+group = "com.okta.android"
+version = "0.1.0"
```
Comment (wdawson) [inline library/build.gradle]: you don't need to redefine this. It's already in `"${rootProject.versionName}"` via the `ext` tag

File: library/build.gradle [hunk 3]
```diff
@@ -29,3 +32,24 @@ dependencies {
...
+        desc = "Okta Authentication using AppAuth"
+        userOrg = "okta"
+        licenses = ["Apache-2.0"]
+        websiteUrl = "https://github.com/okta/okta-sdk-appauth-android"
+        vcsUrl = "https://github.com/okta/okta-sdk-appauth-android.git"
+        issueTrackerUrl = "https://github.com/okta/okta-sdk-appauth-android/issues"
+        version {
+            name = "0.1.0"
+            desc = "Okta Authentication using AppAuth"
+            vcsTag = "0.1.0"
```
Comment (wdawson) [inline library/build.gradle]: use the value in `"${rootProject.versionName}"` instead of redefining. Same for the `name` above.

**Comment by jmelberg-okta** [comment]
@wdawson I need to read over [grgit](http://ajoberstar.org/grgit/index.html) a little bit more to understand exactly how it works. But this sounds really useful!

**Comment by wdawson** [comment]
Looks good. Squash and please follow the [contributing guidelines](https://github.com/okta/okta-sdk-appauth-android/blob/master/CONTRIBUTING.md) (I'm assuming we're still trying to stick to those)

File: library/build.gradle [hunk 4]
```diff
@@ -29,3 +43,24 @@ dependencies {
...
 apply from: "${rootDir}/gradle/style.gradle"
+
+bintray {
+    user = System.getenv("BINTRAY_USER")
+    key = System.getenv("BINTRAY_KEY")
+    configurations = ["archives"]
+    pkg {
+        repo = "com.okta.android"
+        name = "okta-appauth"
+        desc = "Okta Authentication using AppAuth"
```
Comment (wdawson-okta) [inline library/build.gradle]: @omgitstom was this what you wanted? I thought you had wanted `okta-sdk-appauth-android`

Comment (jmelberg-okta) [reply library/build.gradle]: Difference:
```
compile 'com.okta.android:appauth-android:0.1.0'
```
verses
```
compile 'com.okta.android:okta-sdk-appauth-android:0.1.0'
```

Comment (jmelberg-okta) [reply library/build.gradle]: I'm partial to `appauth-android`, since this has less redundant name spacing.

Comment (jmelberg-okta) [reply library/build.gradle]: Updated

**Comment by jmelberg-okta** [comment]
Naming conventions approved by @omgitstom.

**Pull request merged**